options.hpp

```diff
@@ -45,56 +45,56 @@ struct options_t {
     options_t(int argc, char *argv[]);
     virtual ~options_t();
 
-    std::string prefix; ///< prefix for table names
+    std::string prefix{"planet_osm"}; ///< prefix for table names
     std::shared_ptr<reprojection> projection; ///< SRS of projection
-    bool append; ///< Append to existing data
-    bool slim; ///< In slim mode
-    int cache; ///< Memory usable for cache in MB
-    boost::optional<std::string> tblsmain_index; ///< Pg Tablespace to store indexes on main tables (no default TABLESPACE)
-    boost::optional<std::string> tblsslim_index; ///< Pg Tablespace to store indexes on slim tables (no default TABLESPACE)
-    boost::optional<std::string> tblsmain_data; ///< Pg Tablespace to store main tables (no default TABLESPACE)
-    boost::optional<std::string> tblsslim_data; ///< Pg Tablespace to store slim tables (no default TABLESPACE)
-    std::string style; ///< style file to use
+    bool append = false; ///< Append to existing data
+    bool slim = false; ///< In slim mode
+    int cache = 800; ///< Memory usable for cache in MB
+    boost::optional<std::string> tblsmain_index{boost::none}; ///< Pg Tablespace to store indexes on main tables (no default TABLESPACE)
+    boost::optional<std::string> tblsslim_index{boost::none}; ///< Pg Tablespace to store indexes on slim tables (no default TABLESPACE)
+    boost::optional<std::string> tblsmain_data{boost::none}; ///< Pg Tablespace to store main tables (no default TABLESPACE)
+    boost::optional<std::string> tblsslim_data{boost::none}; ///< Pg Tablespace to store slim tables (no default TABLESPACE)
```

Comment:
Please also adhere to the 80 column limit while you are changing stuff here (i.e. move comments above the members).

Reply:
Fixed.